run.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+function print_usage {
+    echo "Usage: $0 [options] command"
+    echo "command: serve | watch | docker | update | build | clean | test | start"
+    echo "   -h      print this help"
+    echo "   -c | --common-path <dir>"
+    echo "           Use this directory for the common libraries. Needs -r."
+    echo "   -r | --regenerate-common"
+    echo "           Fetch the common libraries, either from a local directory"
+    echo "           specified with -c, or from pypi is -c is omitted"
+}
+
+# Read command-line options
+while [[ $# > 1 ]]; do
+    key="$1"
+    case $key in
+        -h|--help)
+        print_usage
+        exit
+        ;;
+        -c|--common-path)
+        COMMONPATH="$2"
+        shift
+        ;;
+        -r|--regenerate-common)
+        REGENERATE=YES
+        ;;
+        *)
+        echo "Unknown option $1"
+        exit
+        ;;
+    esac
+    shift
+done
+
+COMMAND=$1
+
+case "$COMMAND" in
+    clean)
+        git clean -dfX
```

Comment:
The current Makefile already does a git clean. It only removes the generated files, too.